gratipay/utils/__init__.py

```diff
@@ -149,13 +151,11 @@ def set_cookie(cookies, key, value, expires=None, httponly=True, path='/'):
     cookies[key] = value
     cookie = cookies[key]
     if expires:
-        if isinstance(expires, datetime):
-            pass
-        elif isinstance(expires, timedelta):
+        if isinstance(expires, timedelta):
             expires += utcnow()
-        else:
-            raise TypeError('`expires` should be a `datetime` or `timedelta`')
```

Comment:
Then it's expected to be `bytes` representing a date in the correct format, like `BEGINNING_OF_EPOCH`.